src/main/java/machines/Truck.java

```diff
@@ -0,0 +1,11 @@
+package machines;
+
+public class Truck extends Machine {
+    public void doWork() {
+        System.out.println("Truck has started the job");
+    }
+
+    public void stopWork() {
+        System.out.println("Truck has stopped the job");
```

Comment:
Similarly to the `doWork` method, the message in `stopWork` should be more informative. It should clearly indicate that it is a `Truck` that has stopped working. Consider changing it to 'Truck has finished working'.